R/use_extendr.R

```diff
@@ -152,16 +146,19 @@ extendr_module! {{
     )"
   )
 
-  make_example_wrappers(pkg_name, wrappers_file, extra_items = example_function_wrapper)
+  make_example_wrappers(pkg_name, wrappers_file, extra_items = example_function_wrapper, path = path)
+  write_namespace(pkg_name, path, quiet = quiet)
 
   if (!isTRUE(quiet)) {
-    message(glue("Done.\n\nPlease run `devtools::document()` for changes to take effect.\nAlso update the system requirements in your `DESCRIPTION` file."))
+    cli::cli_alert_success("Finished configuring {.pkg extendr} for package {.pkg {pkg_name}}.")
+    cli::cli_alert_warning("Please update the system requirement in {.file DESCRIPTION} file.")
+    cli::cli_alert_warning("Please run {.fun rextendr::document} for changes to take effect.")
   }
 
   return(invisible(TRUE))
 }
 
-make_example_wrappers <- function(pkg_name, outfile, extra_items = NULL) {
+make_example_wrappers <- function(pkg_name, outfile, extra_items = NULL, quiet = FALSE, path = ".") {
```

Comment:
I would move the path argument further up, certainly before `quiet` and maybe also before `extra_items`. Also, it would be good to add some comments explaining what the function and the various arguments do. For internal functions, I often like to mimic roxygen2 without actually using it, e.g.:

```r
# Function to generate a simple wrapper file if no autogenerated wrappers are available yet
#
# @param pkg_name ...
# @param outfile ...
```
```

Reply:
Ok, will do.